winpr/libwinpr/clipboard/synthetic_file.c

```diff
@@ -404,24 +365,24 @@ static BOOL add_file_to_list(wClipboard* clipboard, const char* local_name,
 	return TRUE;
 }
 
-static const char* get_basename(const char* name)
+static const WCHAR* get_basename(const WCHAR* name)
 {
-	const char* c = name;
-	const char* last_name = name;
+	const WCHAR* c = name;
+	const WCHAR* last_name = name;
 
 	while (*c++)
 	{
-		if (*c == '/')
+		if (*c == L'/')
```

Comment:
`Be careful with `L'somestring'`, windows understands `UCS-2/UTF16` most other systems `UCS4/UTF32`